src/components/PinInput/src/PinInput.vue

```diff
@@ -126,13 +126,13 @@ export default {
 		},
 
 		onInputPin(event, index) {
-			if (!event.data) {
+			const input = event?.data || event?.target?.value;
```

Comment:
Can you help me understand why `event.data` was used in the first place? I'm use to seeing `event.target.value` everywhere, which seems to be the reliable cross-browser standard for accessing an input's value, so if you're gonna be using that you can probably just get rid of the `event.data`.

Reply:
Oh, does it have something to do with people trying to paste the entire code into the first input cell? 🤔

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/InputEvent

Youre right that it makes sense to just get rid of the event.data in this case

Reply:
No - its just that the input event has a `.data` attribute that seems to have cross-browser support which is why its originally what I was using. On Safari it doesnt seem to be there for some reason, even though the docs say it supports  it. I also tried using `input.native` to see if for some reason that was messing with anything but it had the same effect

Reply:
re: pasting the entire code into the first input cell - the component already supports that functionality 👍

Reply:
great, thanks for the context! i've resolved the open threads and approved the pr